plugin/parser/differ/mysql/differ.go

```diff
@@ -52,6 +52,10 @@ func (*SchemaDiffer) SchemaDiff(oldStmt, newStmt string) (string, error) {
 
 	var newNodeList []ast.Node
 	var inplaceUpdate []ast.Node
+	// inplaceDropNodeList and inplaceAddNodeList are used to handle destructive node updates.
+	// For example, we should drop the old index named 'id_idx' and then add a new index named 'id_idx' in the same table.
+	var inplaceDropNodeList [][]ast.Node
+	var inplaceAddNodeList [][]ast.Node
```

Comment:
qq: any reason we need to use a matrix of ast.Node for inplaceDropNodeList/inplaceAddNodeList/dropNodeList?

Reply:
We may use it to handle the dependency between the node in one statement. But no need for now. Remove it.